frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/noobaa.obcTest.scenario.spec.ts

```diff
@@ -0,0 +1,284 @@
+import {
+  BOUND,
+  MINUTE,
+  OBC_NAME,
+  OBC_RESOURCE_PATH,
+  OBC_RESOURCE_PATH,
+  OBC_STORAGE_CLASS,
+  OBC_STORAGE_CLASS_EXACT,
+  SECOND,
+} from '../../../noobaa-storage-plugin/integration-tests/utils/consts';
+import {
+  testDeployment,
+  deployment,
+} from '../../../noobaa-storage-plugin/integration-tests/mocks/deploymentData';
+import { testName } from '../../../integration-tests-cypress/support/index';
+import { projectNameSpace } from '../../../dev-console/integration-tests/support/pages/app';
+
+const MASK = '•••••';
+const NOOBAA_LABEL = 'app=noobaa';
+const NO_ANNOTATIONS = '0 annotations';
+
+const appHost = `${process.env.BRIDGE_BASE_ADDRESS || 'http://localhost:9000'}${(
+  process.env.BRIDGE_BASE_PATH || '/'
+).replace(/\/$/, '')}`;
+
+class CreateOBCHandler {
```

Comment:
Create a view file and move this class over there. We don't need it in the test specs.